topics/fair/tutorials/fair-clinical/tutorial.md

```diff
@@ -0,0 +1,283 @@
+
+Here are links to two registry projects.  Both websites describe the projects, the aims and the outputs of the studies.
+
+[UK Sjögren's registry](https://www.sjogrensregistry.org/index.php)  
+[UK JIA Biologics Registers](https://sites.manchester.ac.uk/bcrdbspar/) 
+
+## Electronic health records (EHR)
+These are generally the data collected when a person has contact with the health services.  In the UK, EHR data is either at the Trust level or at the country level, e.g. NHS England, and it is usually necessary to specify primary care (GP, pharmacy, dental and optometry) and/or secondary care (hospital and specialists) data.  This data is usually received as anonymised data, to avoid the ability to identify individuals from the data.
+
+[comment]: <> (This part may need rewording. My understanding is that EHRs are provided by: GPs where two main platforms are EMIS Web and TPP SystmOne and; NHS trusts with many different suppliers.  NHS England provides advice on accredited suppliers.)
+[comment]: <> (to do)
+
+In the UK, Clinical Practice Research Datalink ([CPRD](https://cprd.com)) collects anonymised patient data from a network of GP practices links this data to a range of other health-related data and provides a longitudinal, representative UK population health dataset. 
+
+The [Health Data Research Innovation Gateway](https://www.healthdatagateway.org/) can help researchers to find and discover UK health datasets, tools and other resources to further health research.
+
+> <hands-on-title>Explore HDRUK Innovation Gateway for datasets - {% icon time %} 5 minutes</hands-on-title>
+> 
+> 1. Click on Datasets in the [Health Data Research Innovation Gateway](https://www.healthdatagateway.org/)
+> 
+> 2. In the filters, expand Publisher and in Search Filter type CPRD and enter  
+> 
+> 3. Select CPRD  
+> 
+> 4. Expand Phenotype and search for rheumatoid, and select Rheumatoid Arthritis  
+> 
+> 5. Take a look at the information about the two datasets  
+> 
+> 6. Clear the filters, and search for datasets related to your areas of interest  
+>
+{: .hands_on}
+
+# The importance of making clinical datasets FAIR
+
+## Associations between clinical features and omics
+In some recent work in rheumatoid arthritis [The RA-MAP Consortium, 2022](https://www.nature.com/articles/s41597-022-01264-y), it was demonstrated that simple patient demographics such as sex, ethnicity and age are all drivers of expression variation in addition to disease activity.  Principal component driver plots highlighted critical associations between diverse clinical features and omics.  This shows how rich clinical information may be key to analysis in some, if not, many diseases.  
+
+![RA-MAP.](../../images/RAMAP_graphic.png)
```

Comment:
It looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details. *This comment may apply to more images, please consider checking all of them.*